packages/cli/src/util/errors.ts

```diff
@@ -2,3 +2,12 @@
  * Expected error that shouldn't print a stack trace
  */
 export class YargsError extends Error {}
+
+export type Result<T> = {err: null; result: T} | {err: Error};
+export async function wrapError<T>(promise: Promise<T>): Promise<Result<T>> {
```

Comment:
This function already exists as part of beacon node utils
https://github.com/ChainSafe/lodestar/blob/c423687aa95c428fc3c781dc92129a8b3074babb/packages/beacon-node/src/util/wrapError.ts#L16

Would make sense to move this to @lodestar/utils to make it more reusable across packages